src/Halogen/Store/Monad.purs

```diff
@@ -191,12 +182,60 @@ runStoreT
   -> (s -> a -> s)
   -> H.Component q i o (StoreT a s m)
   -> Aff (H.Component q i o m)
-runStoreT initialStore reducer component = do
+runStoreT initialStore reducer component =
+  _.root <$> runAndEmitStoreT initialStore reducer component
+
+-- | Run a component in the `StoreT` monad.
+-- |
+-- | Requires an initial value for the store, `s`, and a reducer that updates
+-- | the store in response to an action, `a`.
+-- |
+-- | This can be used directly on the root component of your application to
+-- | produce a component that Halogen can run, so long as the base monad can
+-- | be fixed to `Aff`.
+-- |
+-- | Returns a component that can be run with `runUI` and an emitter with can
+-- | be used to react to store changes. This can be used, for example, to
+-- | persist parts of the store to local storage or some other persistence
+-- | mechanism, allowing you to push these concerns to the boundaries of the
+-- | application, outside of the component.
+-- |
+-- | ```purs
+-- | main = do
+-- |   -- load initial store values from local storage.
+-- |   field1 <- LocalStorage.getItem "field1"
+-- |   ...
+-- |   fieldN <- LocalStorage.getItem "fieldN"
+-- |   let initialStore = mkStore field1 ... fieldN
+-- |   launchAff_ do
+-- |     body <- Halogen.Aff.awaitBody
+-- |     { emitter, root } <- runAndEmitStoreT initialStore reducer rootComponent
+-- |     runUI root unit body
+-- |     let selectField1 = selectEq _.field1
+-- |     let selectFieldN = selectEq _.fieldN
+-- |     let subscribe_ = void <<< HS.subscribe
+-- |     liftEffect do
+-- |       -- save new store values to local storage as they change.
+-- |       subscribe_ $ selectEmitter selectField1 emitter $ LocalStorage.setItem "field1"
+-- |       ...
+-- |       subscribe_ $ selectEmitter selectFieldN emitter $ LocalStorage.setItem "fieldN"
```

Comment:
I'd like to trim this documentation comment down a little bit by just indicating a single local storage field being updated via the store. Thank you for writing the example — it's just a little long, and I think the concept is illustrated just as well with only one field.

```suggestion
-- |   -- load initial store values from local storage.
-- |   field <- LocalStorage.getItem "field"
-- |   let initialStore = mkStore field
-- |   launchAff_ do
-- |     body <- Halogen.Aff.awaitBody
-- |     { emitter, component } <- runAndEmitStoreT initialStore reducer rootComponent
-- |     runUI component unit body
-- |     let selectField = selectEq _.field
-- |     liftEffect do
-- |       -- save new store values to local storage as they change
-- |       void $ H.subscribe $ selectEmitter selectField emitter $ LocalStorage.setItem "field"
```